src/workloads/replication/startup/1_2_5GB_ddl.yml

```diff
@@ -0,0 +1,34 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: >
+  Adds DDL operations to be replayed during startup recovery for the light phase.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2500
+
+NumPhases: &max_phase 1
+Database: &database "startup_5GB_ddl"
+CollectionCount: &collectionCount 5
+
+Actors:
+- StopCheckpointing:
+  LoadConfig:
+    Path: "../../../phases/replication/startup/StartupPhasesTemplate.yml"
+    Key: StopCheckpointTemplate
+    Parameters:
+      active: [0]
+      nopInPhasesUpTo: *max_phase
+
+- DDLAndCrudOperations:
```

Comment:
Am I missing something? Why are these DDL operations if the previous usages of InsertDataTemplate are not?

Reply:
Thanks for the clarification.